test/controllers/opensearch_controller_test.exs

```diff
@@ -0,0 +1,12 @@
+defmodule HexWeb.OpensearchControllerTest do
+  use HexWeb.ConnCase, async: true
+
+  test "opensearch" do
+    conn = get build_conn(), "/hexsearch.xml"
+
+    path          = Path.join([__DIR__, "..", "fixtures"])
+    expected_body = File.read!(Path.join(path, "opensearch.xml"))
+
+    assert response(conn, 200) =~ expected_body
```

Comment:
We should only match against a subset. There is no need to include a fixture that is an exact copy of the expected output. We can match against something like: `<Url type="text/html" method="get" template="http://localhost:4001/packages?search={searchTerms}&amp;sort=downloads" />`.